content/en/observability_pipelines/quickstart/_index.md

```diff
@@ -0,0 +1,170 @@
+
+Most customers have a common tools/security cluster for their environments, which all other clusters are connected to.
+This is a natural spot to deploy OP, as connectivity issues have already been worked out. **Other, more complex setups will be covered in other guides.**
+
+## Installing OP
+### Starting Configuration
+The following Helm configurations are pre-assembled for the major cloud providers that we support.
+{{< tabs >}}
+{{% tab "AWS EKS" %}}
+[Helm Chart](/resources/yaml/observability_pipelines/quickstart/aws_eks.yaml)
+{{% /tab %}}
+{{% tab "Azure AKS" %}}
+[Helm Chart](/resources/yaml/observability_pipelines/quickstart/azure_aks.yaml)
+{{% /tab %}}
+{{% tab "Google GKE" %}}
+[Helm Chart](/resources/yaml/observability_pipelines/quickstart/google_gke.yaml)
+{{% /tab %}}
+{{< /tabs >}}
+
+After downloading the correct Helm chart for your provider, you should replace the `datadog.apiKey` and `datadog.configKey` values to match your pipeline. Then you can install it into your cluster with the following commands:
+
+```
+$ helm repo add datadog https://helm.datadoghq.com
+$ helm repo update
+$ helm update --install \
+    opw datadog/observability-pipelines-worker \
+    -f <HELM CHART YOU DOWNLOADED>
+```
+
+### Load Balancing
+Our preference is to defer to the load balancers provided by your cloud- they will be higher-performance than anything we could offer ourselves,
+and they will properly react to autoscaling events that the default Helm setup is configured for. All these load balancers are Internal-facing,
+so they will only be accessible inside of your network.
+
+You will enter the load balancer URL given to you by Helm when it comes time to configure the Datadog Agent.
+
+#### Cross-AZ Load Balancing
+The provided Helm configurations do their best to simplify load balancing, but you must take into consideration the potential price implications of cross-AZ traffic. **Wherever possible, the samples try to avoid creating situations where multiple cross-AZ hops can happen, preferring performance, simplicity, and cost optimizations at the potential expense of fault tolerance.** Links will be provided to the cloud provider info pages for more information on settings you can use to change that tradeoff however you would like for your own deployments.
+
+{{< tabs >}}
+{{% tab "AWS EKS" %}}
+On AWS, we use NLBs provisioned by the [AWS Load Balancer Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/). It allows
+much more flexibility and correctness than the in-tree controller that AWS provides.
+
+The sample configurations do not enable the "cross-zone load balancing" feature available in this controller; to change this, adding the following annotation to the `service` block should suffice:
+
+```
+  service.beta.kubernetes.io/aws-load-balancer-attributes: load_balancing.cross_zone.enabled=true
```

Comment:
```suggestion
    service.beta.kubernetes.io/aws-load-balancer-attributes: load_balancing.cross_zone.enabled=true
```

If you're trying to indent to where the `values.yaml` would have it I think this is correct. Otherwise maybe just left aligned?